app/controllers/v6/assessments_controller.rb

```diff
@@ -48,7 +48,23 @@ def create
 
     def dependants(input_params, submission_date)
       dependant_params = input_params.fetch(:dependants, [])
-      dependant_params.map { |p| Dependant.new(p.merge(submission_date:)) }
+      dependant_params.map do |params|
+        updated_params = convert_monthly_income_params(params)
+        Dependant.new(updated_params.merge(submission_date:))
+      end
+    end
+
+    def convert_monthly_income_params(params)
+      income = params[:income]
+      monthly_income = params[:monthly_income]
+      if income.present?
+        income_amount = income[:amount]
+        income_frequency = income[:frequency]
+      elsif monthly_income.present?
+        income_amount = monthly_income
+        income_frequency = CFEConstants::MONTHLY_FREQUENCY
+      end
+      params.except(:monthly_income, :income).merge(income_amount:, income_frequency:)
```

Comment:
Feels like we're missing a test. If neither items are supplied (which seems likely right now by CCQ) then amount and frequency will both be nil (rather than the default 0 which is never used)